pipeline/api/cloud.py

```diff
@@ -55,9 +55,19 @@ class PipelineCloud:
     token: Optional[str]
     url: Optional[str]
 
-    def __init__(self, url: str = None, token: str = None) -> None:
+    def __init__(
+        self,
+        *,
+        url: str = None,
+        token: str = None,
+        timeout=60.0,
```

Comment:
it's probably worth adding timeout to the sync client methods too to make things consistent, otherwise this is probably going to confuse the end user if they're specifying the timeout and it does nothing

Reply:
I made a note of this before, done now